### ISSUE TITLE
Sharp compatibility issue with some metamod plugins

### ISSUE THREAD

**Issue by rcon420** [roflmuffin/CounterStrikeSharp#86]
could this bug be fixed?  Because of cs2sharp some metamod plugins  commands show duplicated messages in the chat.
en example is Wills Metamod Surf Timer.

**Comment by KillStr3aK** [comment]
duplicate of #34

**Comment by xen-000** [comment]
The duplicated messages are caused by `player_chat` being fired from multiple plugins, namely CS2Fixes and CSSharp, and the surf timer uses lua vscript so it relies on `player_chat` for commands. This is pretty much the only conflict between both plugins as CS2Fixes now uses a different method to hook chat messages.

**Comment by roflmuffin** [comment]
I think if you are running cs2fixes, cssharp and will's surf timer you are somewhat deserving of issues. If you want to run wills, I would run it either with CS2fixes or cssharp but not both

**Comment by ws-cs2** [comment]
> I think if you are running cs2fixes, cssharp and will's surf timer you are somewhat deserving of issues. If you want to run wills, I would run it either with CS2fixes or cssharp but not both

Hopefully we can work this one out, we're going to move to be dependent on cs2fixes (any any future movement plugins will be too) when it hits 1.0.0 and lots of servers would still like to use counterstrikesharp!

Reason being is Xen has coded a trigger_push fix and any other fixes we will need for movement game modes will go into there (and we need MovementUnlocker & LuaUnlocker)

I'm sorry for causing issues by using player_chat 😄

**Comment by ws-cs2** [comment]
I'll have a go at deduping them on my side then we should all be compatible

**Comment by roflmuffin** [comment]
A lot of people using a plugin that is declared not ready for use :P

**Comment by roflmuffin** [comment]
I am closing this issue and creating a more over-arching ticket that encompasses CS2 fixes compatibility.
